integration-cli/docker_cli_pull_local_test.go

```diff
@@ -486,7 +486,7 @@ func (s *DockerRegistrySuite) TestRunImplicitPullWithNoTag(c *check.C) {
 	c.Assert(out, checker.Contains, fmt.Sprintf("Unable to find image '%s:latest' locally", repo))
 
 	// There should be only one line for repo, the one with repo:latest
-	outImageCmd, _, err := dockerCmdWithError("images", repo)
+	outImageCmd, _, _ := dockerCmdWithError("images", repo)
```

Comment:
Hum we could even use `dockerCmd(c, "images", repo)` here so no need to check errors 👼